sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -216,6 +288,42 @@ public partial class ClassifyDocumentResultCollection : System.Collections.Objec
         public string ProjectName { get { throw null; } }
         public Azure.AI.TextAnalytics.TextDocumentBatchStatistics Statistics { get { throw null; } }
     }
+    public partial class CurrencyResolution : Azure.AI.TextAnalytics.BaseResolution
+    {
+        internal CurrencyResolution() { }
+        public string Iso4217 { get { throw null; } }
+        public string Unit { get { throw null; } }
+        public double Value { get { throw null; } }
+    }
+    public partial class DateTimeResolution : Azure.AI.TextAnalytics.BaseResolution
+    {
+        internal DateTimeResolution() { }
+        public Azure.AI.TextAnalytics.DateTimeSubKind DateTimeSubKind { get { throw null; } }
+        public Azure.AI.TextAnalytics.TemporalModifier? Modifier { get { throw null; } }
+        public string Timex { get { throw null; } }
```

Comment:
Silly question but I'm assuming the `x` comes from `extended` or something of the sort, and `Timex` is not a typo. Is that right?

Reply:
Correct, it's not a typo; this is what the service team called it in their swagger. That being said, I'm personally expecting that we will give it a better name in the future as we prepare it for an arch review.